client/modules/IDE/components/EditorShowHint.jsx

```diff
@@ -0,0 +1,524 @@
+/* eslint-disable */
```

Comment:
This file doesn't use JSX syntax so I think its extension should be just .`js`. I think you could call this file `show-hint.js` so it's obvious the CodeMirror file it is modified from.

Also, I was comparing your modified version with the CodeMirror version (see [diff](https://www.diffchecker.com/yAdquIjS) and I noticed that you removed a lot of the `aria-` attributes. These are really important to have for screen-reader users, is it possible to add them back with your UI changes?

Reply:
I don't remember removing them. Is it possible that you were comparing to a different version of their original file? You can see that all `aria` related lines miss the semicolon, which indicates they might add them later? But yeah, they are good, so I added them to the file. The file was also renamed.